src/js/tech/tech.js

```diff
@@ -437,6 +437,15 @@ class Tech extends Component {
     return '';
   }
 
+  static isTech(component) {
+    /* jshint proto:true */
+    let proto = component.__proto__;
+    /* jshint proto:false */
+    if (Object.getPrototypeOf) {
```

Comment:
Hmm, that seems that it could be useful. Is there a synonymous `isComponent` method? If not, that might be useful...